src/api/goods.js

```diff
@@ -1,11 +1,25 @@
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
-export function getAll() {
-  return fetch(API_URL)
-    .then(response => response.json());
-}
+export const getResponse = () => fetch(API_URL)
+  .then(response => response.json());
 
-export const get5First = () => {};
+export const get5First = () => fetch(API_URL)
+  .then(response => response.json())
+  .then((goodsList) => {
+    const preparedList = goodsList
+      .sort((firstGood, secondGood) => (
+        firstGood.name.localeCompare(secondGood.name)
+      ))
+      .slice(0, 5);
```

Comment:
No need to create a variable if you return it right away:
```suggestion
  .then((goodsList) => (
    goodsList
      .sort((firstGood, secondGood) => (
        firstGood.name.localeCompare(secondGood.name)
      ))
      .slice(0, 5));
```